packages/cli/babel/index.js

```diff
@@ -1,3 +1,5 @@
+// Is this file used/required? If not, can it be removed?
```

Comment:
Probably a question for @rschristian 

I couldn't find any instance of this file being used. Unless it's used implicitly by Babel, could we remove it entirely?

Reply:
Not used by us, no. However, users can add it as a preset to their own babel config if they so wish. 

It's not a great setup tbh, so I'm going to try to remove it in the next major, but there are some users who depend on it for now.